### ISSUE TITLE
some diseases > 100% score match

### ISSUE THREAD

**Issue by nlwashington** [monarch-initiative/monarch-legacy#605]
For this disease:
https://monarchinitiative.org/disease/OMIM:608902

the top matching hits for this one are all > 100%. 

this is all very odd, considering there is only one phenotype, and it very loosely matches the disease itself (generally they all match on neoplasm/cancer).  (also, really, why is neoplasm the only phenotype for 'Drug Metabolism, Poor, Cyp2d6-related'...and why is it a disease anyway?  should a disease be a disease with one phenotype?)

this is presumably some logic that needs to be fixed on the owlsim side.

**Comment by jmcmurry** [comment]
@nlwashington is this still an issue? We no longer show the scores in disease pages so not sure...

**Comment by nlwashington** [comment]
yes, the score is still an issue... it is visible in the phenogrid, and shows up as 137%.  this will need to be investigated from the owlsim side of things.

**Comment by jmcmurry** [comment]
@nlwashington  I looked at all tiers for the example you provided and can no longer see the 137% in phenogrid. Has this been addressed?

**Comment by jmcmurry** [comment]
if it was previously addressed this error is now back
@kshefchek ?

<img width="635" alt="screen shot 2016-08-09 at 11 59 14 pm" src="https://cloud.githubusercontent.com/assets/3658863/17544633/51ed9e4e-5e8d-11e6-9c93-1390083c1512.png">
